<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientTCPIPClass</class>
 <widget class="QMainWindow" name="ClientTCPIPClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client </string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushBtnConnect">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Connexion</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelStatus">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditIP">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPort">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelIP">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IP :</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPort">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Port :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_error">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnCelsius">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Demander la température en celsius</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnFahrenheit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>110</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Demander la température en fahrenheit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnHygrometie">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>160</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Demander l'hygrométrie</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCapteur">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>N° Capteur </string>
    </property>
   </widget>
   <widget class="QSpinBox" name="numcapteur">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>505</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ClientTCPIP.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushBtnConnect</sender>
   <signal>clicked()</signal>
   <receiver>ClientTCPIPClass</receiver>
   <slot>onConnectButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushBtnCelsius</sender>
   <signal>clicked()</signal>
   <receiver>ClientTCPIPClass</receiver>
   <slot>onSendCelsiusButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushBtnFahrenheit</sender>
   <signal>clicked()</signal>
   <receiver>ClientTCPIPClass</receiver>
   <slot>onSendFarhenheitButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushBtnHygrometie</sender>
   <signal>clicked()</signal>
   <receiver>ClientTCPIPClass</receiver>
   <slot>onSendHygrometrieButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onConnectButtonClicked()</slot>
  <slot>onSendMessageButtonClicked()</slot>
  <slot>onSendCelsiusButtonClicked()</slot>
  <slot>onSendFarhenheitButtonClicked()</slot>
  <slot>onSendHygrometrieButtonClicked()</slot>
 </slots>
</ui>
